### PR TITLE
CC3XX: Update mbedtls hash entrypoints

### PR DESCRIPTION
**Pull request by salomethirot-arm**

### CODE REVIEW

File: library/psa_crypto_driver_wrappers.c [hunk 1]
```diff
@@ -44,6 +44,18 @@
...
+
+#if defined(PSA_CRYPTO_DRIVER_CC3XX)
+#ifndef PSA_CRYPTO_DRIVER_PRESENT
+#define PSA_CRYPTO_DRIVER_PRESENT
+#endif
+#ifndef PSA_CRYPTO_ACCELERATOR_DRIVER_PRESENT
+#define PSA_CRYPTO_ACCELERATOR_DRIVER_PRESENT
+#endif
+/* Change the path to fit your machine */
+#include "<path>/trusted-firmware-m/lib/ext/cryptocell-312-runtime/codesafe/src/psa_driver_api/include/cc3xx.h"
```
Comment (noonfom) [inline library/psa_crypto_driver_wrappers.c:56]:
```suggestion
#include "cc3xx.h"
```

However, we need the following in the CMakeListst.txt under the PSA driver API folder in TF-M:

```CMake
target_include_directories(${CC312_LIB_TARGET}
        PUBLIC
        include
        .
)
```

**Comment by noonfom** [comment]
@salomethirot-arm I want to merge this but I don't see a `cc3xx.h` file in your [patch](https://review.trustedfirmware.org/c/TF-M/trusted-firmware-m/+/10607). How is `psa_crypto_driver_wrappers.c` getting access to the hash multipart prototypes?

**Comment by salomethirot-arm** [comment]
@noonfom I realized I forgot to commit the file, will fix that asap and then you can merge

**Review by noonfom — APPROVED** [review]
LGTM

**Pull request merged**